MagnaLorem/CampusAlly
# Test : chat rooms

How to test

Open up localhost in 2 different tabs(windows).

test to see if you can retrieve the messages if you are in the same chat rooms.
test to see if you won't see the messages if you are in the different chat rooms.
